tests/Controllers/FeedControllerTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Moq;
+using Phytime.Controllers;
+using Phytime.Models;
+using System.Collections.Generic;
+using Xunit;
+using Microsoft.Extensions.Configuration;
+using System.ServiceModel.Syndication;
+
+namespace UnitTestApp.Tests.Controllers
+{
+    public class FeedControllerTests
+    {
+        private const string SourceUrl = "https://psyjournals.ru/rss/psyedu.rss";
+        private const int SourceUrlItemsCount = 7;
+
+        [Fact]
+        public void GetSyndicationItemsTest()
+        {
+            var mock = new Mock<IConfiguration>();
+            var mockRep = new Mock<IRepository>();
+            FeedController controller = new FeedController(mock.Object, mockRep.Object);
+            int result = controller.GetSyndicationItems(SourceUrl).Count;
+            Assert.Equal(SourceUrlItemsCount, result);
+        }
+
+        [Fact]
+        public void FormFeedTest()
```

Comment:
https://dzone.com/articles/7-popular-unit-test-naming

choose which you like and rename